### PR TITLE
[zh] Sync #14307 advanced rate limit example into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync #14307 advanced rate limit example into Chinese

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/tasks/policy-enforcement/rate-limit/index.md [hunk 1]
```diff
@@ -9,33 +9,34 @@ test: no
...
 
 Envoy 可以为您的网格[设置全局速率限制](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/other_features/global_rate_limiting)。
 Envoy 中的全局速率限制使用 gRPC API 向速率限制服务请求配额。
 下文使用的[参考实现](https://github.com/envoyproxy/ratelimit)后端，是用 Go
 语言编写并使用 Redis 作为缓存。
 
 1. 参考下面的 ConfigMap [配置限流规则](https://github.com/envoyproxy/ratelimit#configuration)，
-   对 `/productpage` 的限制速率为每分钟 1 次，其他所有请求的限制速率为每分钟 100 次。
+   将 `/productpage` 的限制速率为每分钟 1 次，用于后续高级示例的 `api` 值，
+   并将其他所有请求的限制速率为每分钟 100 次。
```
Comment (windsonsea) [inline content/zh/docs/tasks/policy-enforcement/rate-limit/index.md:39]:
```suggestion
   并将其他所有请求的速率限制为每分钟 100 次。
```
这样更通顺一些

File: content/zh/docs/tasks/policy-enforcement/rate-limit/index.md [hunk 2]
```diff
@@ -150,6 +156,91 @@ Envoy 中的全局速率限制使用 gRPC API 向速率限制服务请求配额
     EOF
     {{< /text >}}
 
+### 全局速率限制高级示例  {#global-rate-limit-advanced-case}
+
+此示例使用正则表达式来匹配 `/api/*` `uri`
+并使用 VirtualService http 名称定义在路由级别插入的速率限制操作。
```
Comment (windsonsea) [inline content/zh/docs/tasks/policy-enforcement/rate-limit/index.md:162]:
```suggestion
并使用 VirtualService http 名称来定义在路由级别插入的速率限制操作。
```

File: content/zh/docs/tasks/policy-enforcement/rate-limit/index.md [hunk 3]
```diff
@@ -150,6 +156,91 @@ Envoy 中的全局速率限制使用 gRPC API 向速率限制服务请求配额
...
+前面示例中插入的 PATH 值 `api` 开始发挥作用。
+
+1. 更改 VirtualService，前缀 `/api/v1/products` 被移动到名为 `api` 的路由中：
+
+{{< text bash >}}
```
Comment (windsonsea) [inline content/zh/docs/tasks/policy-enforcement/rate-limit/index.md:167]: The original code block indents incorrectly.

File: content/zh/docs/tasks/policy-enforcement/rate-limit/index.md [hunk 4]
```diff
@@ -150,6 +156,91 @@ Envoy 中的全局速率限制使用 gRPC API 向速率限制服务请求配额
...
+    route:
+    - destination:
+        host: productpage
+        port:
+          number: 9080
+    name: api
+EOF
+{{< /text >}}
+
+1. 应用 EnvoyFilter 在任意 1 到 99 产品中的路由级别添加速率限制操作：
```
Comment (windsonsea) [inline content/zh/docs/tasks/policy-enforcement/rate-limit/index.md:205]:
```suggestion
1. 应用 EnvoyFilter 在结果为 1 到 99 的任一路由级别添加速率限制操作：
```

**Review by windsonsea — COMMENTED** [review]
Here are some text suggestions for your reference.

File: content/zh/docs/tasks/policy-enforcement/rate-limit/index.md (hunk 3 above)
Comment (wilsonwu) [reply content/zh/docs/tasks/policy-enforcement/rate-limit/index.md:167]: Fixed.

**Review by windsonsea — APPROVED** [review]
lgtm, thanks.

**Pull request merged**